test/cmdlineTests/yul_string_format_ascii/output.json

```diff
@@ -107,6 +107,15 @@ object \"C_10\" {
                 }
             }
 
+            function extract_byte_array_length(data) -> length {
```

Comment:
It is a bit weird that this function is added but it is not used anywhere...

Reply:
It's because of this: https://github.com/ethereum/solidity/pull/8898/files#r425307721

Reply:
I added checks for call only when isByteArray, but didn't trigger change here locally. Will investigate

Reply:
Fixed now